db/config.cc

```diff
@@ -1083,6 +1083,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , alternator_ttl_period_in_seconds(this, "alternator_ttl_period_in_seconds", value_status::Used,
         60*60*24,
         "The default period for Alternator's expiration scan. Alternator attempts to scan every table within that period.")
+    , alternator_listen_backlog(this, "alternator_listen_backlog", value_status::Used, 100, "The listen_backlog of Alternator http server, increase this parameter value in scenarios where there are a large number of connections.")
```

Comment:
I think this description is a bit misleading: The problem requiring large listen backlog isn't a large number of connections being active at the same time, but an application trying to create a large number of connections at the same time, because Scylla had a chance to accept() them. Since Scylla is quite diligent in accepting new connections, I'm curious how or where you encountered this problem.